src/Directory.Build.props

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why did you change this to a props file? This was intentionally a targets file because of the order of evaluation. Props files get evaluated before items in the project file, and targets files get evaluated after the project file. There items in here that need the project file to get a chance to add values first.